barman/backup_executor.py

```diff
@@ -1825,21 +1825,19 @@ def check(self, check_strategy):
         :param CheckStrategy check_strategy: the strategy for the management
              of the results of the various checks
         """
-        check_strategy.init_check("pgespresso extension")
+        check_strategy.init_check("postgres minimal version")
         try:
             # We execute this check only if the postgres connection is non None
```

Comment:
We should rewrite this comment and the docstring above since the meaning of this check has changed from "check whether postgres supports concurrent backup" to "check whether Barman supports this version of postgres.

Reply:
Changed